components/event-sources/adapter/http/README.md

```diff
@@ -0,0 +1,74 @@
+# HTTP Adapter Source
+
+The http adapter is a http server which receives [CloudEvents](https://github.com/cloudevents/spec) in version 1.0 and proxies them to a preconfigured sink.
+The adapter is written with the [cloudevents sdk-go](https://github.com/cloudevents/sdk-go) and is fully compatible with CloudEvents specification.
+
+## Specification
+
+The http adapter is CloudEvents 1.0 compatible. It accepts binary as well as structured content mode. Only batched mode is not implemented.
+Read more about the content types and the http protocol binding [here](https://github.com/cloudevents/spec/blob/master/http-protocol-binding.md#13-content-modes).
+
+## Run
+
```

Comment:
```suggestion
### Run the adapter locally
```
I would have two subsections to the `Usage` part:
- `Run locally`
- `Send events to adapter`